spack/package.py

```diff
@@ -69,14 +69,14 @@ class Arbor(CMakePackage, CudaPackage):
 
     # python (bindings)
     extends("python", when="+python")
-    depends_on("python@3.6:", when="+python", type=("build", "run"))
+    depends_on("python@3.7:", when="+python", type=("build", "run"))
     depends_on("py-numpy", when="+python", type=("build", "run"))
     with when("+python"):
         depends_on("py-pybind11@2.6:", type=("build", "run"))
```

Comment:
This this weird. Do we depend on pybind11 2.6 or 2.8? Also, isn't pybind11 header-only?

Reply:
1. The second pybind line has the `when` option: there was a breaking change in pybind.
2. Yes, but that does not mean it's not a build-dep that Spack needs to supply? The submodules are not used for Spack builds. Perhaps you mean `'run'`? Yes, that could probably be removed!